src/main/python/rlbot/utils/rendering/rendering_manager.py

```diff
@@ -97,6 +97,24 @@ def draw_line_2d(self, x1, y1, x2, y2, color):
         self.render_list.append(message)
         return self
 
+    def draw_polyline_2d(self, vec, color):
```

Comment:
I liked the C# param name `vectors`, would you mind using it here too? vec is potentially misleading.

Reply:
https://github.com/RLBot/RLBot/blob/05b6b734ac5936634bb22ec4ca36e967170d9382/src/main/python/rlbot/utils/rendering/rendering_manager.py#L100-L103

Parameter was renamed.